.github/workflows/ci-cd-pims-master.yml

```diff
@@ -81,6 +81,28 @@ jobs:
         run: |
           ./openshift/4.0/player.sh build api -apply
 
+  tag-images:
```

Comment:
uat hasn't been tagging its images, not a big deal, but it should be doing this going forward.